forum/qa-include/qa-pw-blocked-list.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+require_once QA_INCLUDE_DIR . 'pages/pw-blocked-list.php';
```

Comment:
brak nowej linii oraz to powinno być częścią pluginu

Reply:
jw